app/Livewire/Home/TrendingHashtags.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Livewire\Home;
+
+use Livewire\Component;
+use App\Queries\Feeds\TrendingHashtags as TrendingHashtagsQuery;
+
+class TrendingHashtags extends Component
```

Comment:
Instead of doing this; can you return the query results directly from the controller itself, and also, can you add tests for this?

Reply:
Yes. I will do that and update it.